cms/static/js/certificates/factories/certificates_page_factory.js

```diff
@@ -0,0 +1,31 @@
+// Backbone.js Page Object Factory: Certificates
+
+/**
+Notes from Andy Armstrong:
+The basic idea of a page factory is that it is a single RequireJS dependency that can be loaded in a template
+to create a page object.  This was added for the RequireJS Optimizer, which needs to have a single root to determine
+statically all of the dependencies needed by a page.  The RequireJS Optimizer combines these dependencies into a single
+optimized JS file. Mako templates typically contain a block that constructs the page object using this page factory.
+Unit tests for the page factory verify that it behaves as desired. Some of these factories are more complex than others.
```

Comment:
I do not see (on sandbox) that the new JS files are getting optimized into a single file. Add your factory file to the list in cms/static/build.js.